styles/links.css

```diff
@@ -26,15 +26,11 @@ a {
   --bs-link-decoration: underline dotted var(--underline-color) 1px;
 }
 
-.bg-dark a {
-  --bs-link-color-rgb: 255, 255, 255;
-}
-
 /* This may be unnecessary in Bootstrap 5.4+ https://github.com/twbs/bootstrap/pull/39098 */
 a:not(.btn),
 .btn-link {
   text-decoration: var(--bs-link-decoration);
-  text-underline-position: under;
```

Comment:
I believe this was a deliberate decision on link style.  Why are you changing it here?

Reply:
Because that's what the designs look like. Are link underlines different between the header and body?

Reply:
Reverted per slack discussion.